frontend/src/lobby/GameOptions.jsx

```diff
@@ -106,4 +133,12 @@ Chaos.propTypes = {
   packsNumber: PropTypes.number
 };
 
+const Unavailable = () => (
```

Comment:
Instead of showing that it's not possible to make a decadent sealed, I would block the "sealed" type and hide the decadent mode when you use the type "sealed".

Reply:
Done. https://github.com/dr4fters/dr4ft/pull/990/commits/2e29400d94b9e8cf5f5f39191619410cbae3cc48